File: Snacks 17_10_2025/worldPopulationGrowth.js
let population = 0;
let initialPopulation = 8230000000;
let increasePolulation = 0;
let pupulationX2 = 0;

console.log("Year	Anticipated Population	Increase Population");
for (let count = 1; count <= 75; count++){
	population = 8.23 * (2.71828 ** (0.0084*count));
	increasePolulation = population - initialPopulation;
	console.log(count ,"	",population ,"	 ", increasePolulation);
}

File: Snacks 17_10_2025/world_population_growth.py
#using  P(t) = Po . e^rt
population = 0
initialpopulation = 8230000000
increasePolulation = 0
pupulationX2 = 0

print("Year	Anticipated Population		Population Increase")
for count in range (1, 76):
	population = 8230000000 * (2.71828 ** (0.0084*count))
	increasePolulation = population - initialpopulation
	print(f"{count}	{population}		{increasePolulation}")
